frontend/src/app/commonComponents/Header.tsx

```diff
@@ -40,6 +41,12 @@ const Header = () => {
   const onFacilitySelect = (e: React.FormEvent<HTMLSelectElement>) => {
     const id = (e.target as HTMLSelectElement).value;
     dispatch(updateFacility(facilities.find((f) => f.id === id)));
+
+    // refreshes the page with the new facility id in the url. The remaining path should be the same
+    // since the header exists outside the router, we can't access the path using react router
+    let currentRoute = window.location.pathname.split("/").slice(3).join("/");
```

Comment:
🙈

Reply:
this will be fixed in https://github.com/CDCgov/prime-simplereport/issues/236

Reply:
so if the user changes the facility they are logged in with via the dropdown, it'd update the db? That seems like the better solution -- why does this PR and related issue exist?

Reply:
My understanding is the selected facility will be managed all on the client side. The issue linked is to add a UI facility selector post login so there will no longer be loading of the queue before a facility is selected

Reply:
So this ticket sounds like it should implement the URL plumbing alone then? Is there a need to save the current facility in the db if the user will be required to select the facility when they log in?